### PR TITLE
fix(script): [fix : DOC-36] Use dotenv for Prisma package scripts #523

### LINKED ISSUE #509: [Bug] - Issue with Responsiveness of Signup Now modal
**Description:**

Similar to #507 the _**Signup Now**_ modal at https://documenso.com/pricing seems to be messed up. Please refer to the screenshot below for more reference.

IMG_9073

The expected outcome would be to center the modal with sufficient margins around it keeping in mind the responsiveness of it.

### LINKED ISSUE #523: [DOC-361] Use dotenv for Prisma package scripts
The app should use the "dotenv" package to load the environment variables.

<sub>From [SyncLinear.com](https://synclinear.com) | [DOC-361](https://linear.app/documenso/issue/DOC-361/use-dotenv-for-package-scripts)</sub>

### PR DESCRIPTION
**Pull request by adityadeshlahre**
Fix #523 [DOC-36] Use dotenv for Prisma package scripts

### CODE REVIEW

**Comment by adityadeshlahre** [comment]
**UPDATE**
image

i m able to load the .env from root level using package.json ! :smile: 

but **`npm run prisma:migrate-dev -w @documenso/prisma`** is giving error 

I am trying to figure it out !
Thank You

**Edit :**
Is this is correct out of command **`npm run prisma:migrate-dev -w @documenso/prisma`**

image


**Edit2:**
Its working fine for fresh docker containers

image

accidentally i pushed it in same PR #516 

[https://github.com/documenso/documenso/pull/516/commits/68458b50d2ab76173319286f8fb810613d24f221]

This is commit which fixed this issue !
Thank You :smile:

**Comment by dguyen** [comment]
Hey @adityadeshlahre!

Sorry but issue #509 was closed as it was a duplicate of #507.

Could you please remove all changes that are not related to #523 from this PR? Thanks!

Also as a note for the future, we prefer it when a PR targets a single issue.

File: packages/prisma/helper.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /// <reference types="@documenso/tsconfig/process-env.d.ts" />
+require('dotenv').config();
```
Comment (Mythie) [inline packages/prisma/helper.ts:2]: Shouldn't need this 😄

Comment (adityadeshlahre) [reply packages/prisma/helper.ts:2]: ohk let me edit this ! :smile:

**Comment by adityadeshlahre** [comment]
> Hey @adityadeshlahre!
> 
> Sorry but issue #509 was closed as it was a duplicate of #507.
> 
> Could you please remove all changes that are not related to #523 from this PR? Thanks!
> 
> Also as a note for the future, we prefer it when a PR targets a single issue.

changes has been reverted ! :+1:

**Comment by adityadeshlahre** [comment]
Hey @Mythie and @dguyen just a followup comment !
if any changes required in this issue & PR let me know :smile: 
Thank You !

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks for your contribution! 🎉 

/tip 30

**Comment by anikdhabal** [comment]
@adityadeshlahre I told you before to commit the package-lock.json file. Now the test is failing due to mismatch between the version of the dotenv-cli package specified in  package.json and the version in package-lock.json.

**Comment by Mythie** [comment]
That’s no stress, I’ll do it in a minute 😄

**Comment by anikdhabal** [comment]
> Reference

Let me fix that.

**Comment by anikdhabal** [comment]
> That’s no stress, I’ll do it in a minute 😄
Let me fix that 😄.

**Comment by anikdhabal** [comment]
> That’s no stress, I’ll do it in a minute 😄

I've made a PR 😄.

**Comment by adityadeshlahre** [comment]
> @adityadeshlahre I told you before to commit the package-lock.json file. Now the test is failing due to mismatch between the version of the dotenv-cli package specified in package.json and the version in package-lock.json.

Hey @anikdhabal thanks for raising the PRs ! :smile:

**Pull request merged**